### PR TITLE
nano.rb: update to nano 2.6.0

### PR DESCRIPTION
**Pull request by megahall**
I haven't updated a homebrew package since the new bottle-checksum procedure began. So let me know what I should do to fix the bottle section of this file if anything.

### CODE REVIEW

File: nano.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Nano < Formula
   desc "Free (GNU) replacement for the Pico text editor"
   homepage "http://www.nano-editor.org/"
-  url "http://www.nano-editor.org/dist/v2.5/nano-2.5.3.tar.gz"
-  mirror "ftp://ftp.gnu.org/pub/gnu/nano/nano-2.5.3.tar.gz"
-  sha256 "b2b060129b9feff2d4870d803a441178c96531de9aed144ec0b83bd63ccb12ee"
+  url "http://www.nano-editor.org/dist/v2.6/nano-2.6.0.tar.gz"
+  mirror "ftp://ftp.gnu.org/pub/gnu/nano/nano-2.6.0.tar.gz"
+  sha256 "d517a5fcaa9fa6ceda8a63e22be1d83dd7f4af13703c73b10a48e95a6e1178d3"
   revision 1
```
Comment (dunn) [inline nano.rb]: You can leave the bottle block alone, but this line needs to be removed, thanks!

**Comment by megahall** [comment]
@dunn fixed it, thanks for the quick confirmation

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-dupes/commit/d321aa7b4a76daa39c738b7b8af2e28b885060f6. Thank you for your contribution to Homebrew @megahall; we appreciate it! :smiley_cat:

**Comment by megahall** [comment]
Thanks guys!

**Pull request closed without merging**